pkg/trie/triedb/node_storage.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/gammazero/deque"
 )
 
+var emptyNode = make([]byte, 1)
```

Comment:
is that `emptyNode` var used anywhere else? if no I think you can define the `hashedNullNode` like this: 

```go
var hashedNullNode = common.MustBlake2bHash([]byte{0})
```